ibtk/src/solvers/solver_utilities.cpp

```diff
@@ -13,6 +13,7 @@
 
 /////////////////////////////// INCLUDES /////////////////////////////////////
 
+#include <memory>
 #include "ibtk/solver_utilities.h"
 
```

Comment:
```suggestion
#include "ibtk/solver_utilities.h"

#include <memory>

```

system headers need to come after IBTK (or IBAMR) headers. clang-format will fix this automatically for you.